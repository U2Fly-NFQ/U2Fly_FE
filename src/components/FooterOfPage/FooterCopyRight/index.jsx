import React from 'react'
import './index.scss'
export default function FooterCopyRight() {
  return (
    <div className="footer-copyright wide grid">
      <div className="footer-copyright">
        <p>Copyright Â© 2022 U2Fly Booking Designed by U2FlyTeam</p>
      </div>
    </div>
  )
}
