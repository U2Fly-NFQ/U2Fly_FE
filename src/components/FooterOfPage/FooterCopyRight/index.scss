.footer-copyright {
  margin-top: 20px;

  .footer-copyright {
    text-align: center;
  }
}
