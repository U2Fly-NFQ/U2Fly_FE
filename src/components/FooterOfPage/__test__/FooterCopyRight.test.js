import renderer from 'react-test-renderer'
import FooterCopyRight from '../FooterCopyRight'

describe('FooterCopyRight test', () => {
  it('FooterCopyRight component renders correctly', () => {
    const tree = renderer.create(<FooterCopyRight />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
