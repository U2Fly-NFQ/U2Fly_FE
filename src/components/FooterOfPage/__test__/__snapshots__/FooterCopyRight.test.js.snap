// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterCopyRight search test FooterCopyRight search component renders correctly 1`] = `
<div
  className="footer-copyright  wide grid"
>
  <div
    className="footer-copyright"
  >
    <p>
      Copyright © 2022 All Rights Reserved
    </p>
  </div>
</div>
`;

exports[`FooterCopyRight test FooterCopyRight component renders correctly 1`] = `
<div
  className="footer-copyright  wide grid"
>
  <div
    className="footer-copyright"
  >
    <p>
      Copyright © 2022 All Rights Reserved
    </p>
  </div>
</div>
`;
